samples/test/minio_pipeline_root_two_step_test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Two step v2-compatible pipeline with minio pipeline_root."""
+
+import kfp
+
+from .two_step import two_step_pipeline
```

Comment:
this doesn't need to be a separate test file, you can put this in the same file just as a separate TestCase

Reply:
done